### PR TITLE
Support --options with hyphens

### PR DESCRIPTION
**Pull request by jwfearn**
Motivation: hyphens are faster to type, since they don't require a modifier key like underscore does. In my experience hyphens are more widely used. I looked at the man pages for some common command line utilities (`curl`, `wget`, `node`, `git`, `zip`, `grep`, `bash`) and hyphen spelling is more common.

We have a number of double-dash long options which are currently spelled with underscores. For example:

```shell
$ roku --check_ip $ROKU_DEV_TARGET
$ roku --zip_file my.zip
```

This PR adds support for spelling them with hyphens while maintaining the underscore spellings for backwards compatibility. So the above style still works as well as hyphen-style:

```shell
$ roku --check-ip $ROKU_DEV_TARGET
$ roku --zip-file my.zip
```

Also, the help message lists the hyphen-spelled versions and omits the underscore_spelled versions to reduce the amount of help text.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,7 +132,23 @@ At this point the application should be running through your proxy.
...
 
-# Publishing to PyPi
+# Developing
+
+## Running from source
+
+To switch to a local source version:
+
+```shell
+pip install -e ~/roku-dev-cli
```
Comment (strattonbrazil) [inline README.md]: This depends on where your install is.  I usually run "pip install ."

**Review by strattonbrazil — APPROVED** [review]
This looks fine to me, but I'd like to see more motivations for the change in the description instead of making assumptions.  Are you running into a problem?

File: README.md (hunk 1 above)
Comment (jwfearn) [reply README.md]: reverted to `/path/to/your/local/repos/roku-dev-cli`

File: roku_dev_cli/version.py [hunk 2]
```diff
@@ -1 +1 @@
-__version__ = '0.32.1'
+__version__ = '0.32.2'
```
Comment (sjbarag) [inline roku_dev_cli/version.py]: Given that we're adding new CLI arguments, we should probably make this a minor-version bump] to follow semver guidance: https://semver.org/#summary

**Review by sjbarag — APPROVED** [review]
I agree with @strattonbrazil – the changes look file, but I find myself asking "sure, but why?"  It feels GNU-ish to me, but I can't find an authoritative source to say that `--foo-bar` is preferred over `--foo_bar` in GNU projects.

Would you mind at least throwing a comment in this PR to explain what inspired this change?

That said, I do prefer `--foo-bar` in general, so thanks for doing this 😉

File: roku_dev_cli/version.py (hunk 2 above)
Comment (jwfearn) [reply roku_dev_cli/version.py]: done

**Comment by jwfearn** [comment]
@sjbarag, I update the PR description above to add a motivation explanation.

**Comment by sjbarag** [comment]
> @sjbarag, I update the PR description above to add a motivation explanation.

Thanks!

**Pull request merged**